toqito/channels/realignment.py

```diff
@@ -64,8 +64,6 @@ def realignment(input_mat: np.ndarray, dim: int | list[int] = None) -> np.ndarra
 
     if isinstance(dim, int):
         dim = np.array([int(dim), int(dim_mat[0] / dim)])
-        if np.abs(dim[1] - np.round(dim[1])) >= 2 * dim_mat[0] * eps:
```

Comment:
What was this line originally supposed to check for?

Reply:
This function was adapted from QETLAB, so it was put in from here:

```
https://github.com/nathanieljohnston/QETLAB/blob/4f50dbf6ccd4a71c99d4f1afdcd85e9bfda5e9ac/Realignment.m#L38
```

TBH though, I'm not sure of a way to be able to generate such an example that triggers.